HaxeManual/12-target-details.tex

```diff
@@ -576,6 +576,44 @@ \section{C++}
 \label{target-cpp}
 \state{NoContent}
 
+
+\subsection{Getting started with Haxe/C++}
+\label{target-cpp-getting-started}
+
+To get started with Haxe/C++, create a new folder and save this class as \ic{Main.hx}.
+
+\haxe{assets/HelloCPP.hx}
+
+To compile Haxe to C++ we need two obvious prerequisites installed:
+
+\begin{itemize}
+	\item \href{http://lib.haxe.org/p/hxcpp}{hxcpp haxelib}. This is the support library for the C++ backend of the Haxe compiler.
+	\item \href{http://www.microsoft.com/express/vc/}{MSVC - Microsoft Visual C++}.
```

Comment:
Not sure, it's not easy to provide a link for mac or linux, and on linux since every distribution has a different install command it's even worst.